nmdc_server/crud.py

```diff
@@ -686,6 +686,19 @@ def get_query_for_all_submissions(db: Session):
     return all_submissions
 
 
+def get_query_for_submitted_pending_review(db: Session):
```

Comment:
I'd prefer this be named...

```diff
- def get_query_for_submitted_pending_review(db: Session):
+ def get_query_for_submitted_pending_review_submissions(db: Session):
```

...as a way of indicating that the function can be used to get _submissions_.